home.html

```diff
@@ -0,0 +1,224 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/00fad0ddbb.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <header>
```

Comment:
- [ ] Kindly ensure your header design matches the design given, check on the about link its color should match the Behance design, ensure the design is as similar to the Behance design as possible, check the comparison below:

Behance 
![image](https://user-images.githubusercontent.com/61408860/181728412-3cbbae92-131e-4473-902c-83fdf02cdb46.png)

Your Design 
![image](https://user-images.githubusercontent.com/61408860/181728468-8cd33efc-74ed-4999-b796-46d775671767.png)